### ISSUE TITLE
Improve color contrast on website (a11y)

### ISSUE THREAD

**Issue by pserighelli** [scribe-org/scri.be#9]
### Terms

- [x] I have searched [open and closed design issues](https://github.com/scribe-org/scri.be/issues?q=is%3Aissue+label%3Adesign)
- [x] I agree to follow Scribe's [Code of Conduct](https://github.com/scribe-org/scri.be/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

There's a section on the website where the subheading has insufficient contrast between foreground and background, making it hard to read. This happens on both Light and Dark modes.

Here's my recommendation:

- On light mode, when the background is `--scribe-blue-lighter`, `--distinct-text` should change from `90, 90, 90` with 90% opacity to `0, 0, 0` with 60% opacity.
- On dark mode, when the background is --scribe-blue-darker, `--distinct-text` should change from `150, 150, 150` with 90% opacity to `0, 0, 0` with 85% opacity.

Reference: [Web Content Accessibility Guidelines (WCAG) about color contrast.](https://www.w3.org/TR/WCAG22/#contrast-minimum)

![Image](https://github.com/user-attachments/assets/59cc66cb-275d-47e6-a47f-4ff43968d512)

VISUALIZATION:
![Image](https://github.com/user-attachments/assets/0cb2f9e6-c84f-4061-b8bf-11831736c121)

**Comment by andrewtavis** [comment]
Closing this up, @pserighelli :) Decided that the easiest thing to do for consistency was to reduce the alpha a bit for the distinct text, which is of now is used as hover text. We can refine it a bit more if need be, but I thought it was important that it be consistent for dark mode such that the text be a shade of white :) Thanks for deducing the problem and making the issue! 🚀
